packages/hebao_v1/contracts/stores/Data.sol

```diff
@@ -7,9 +7,9 @@ library Data
 {
     struct Guardian
     {
-        address  addr;
-        uint     group;
-        uint128  validSince;
-        uint128  validUntil;
+        address addr;
+        uint64  group;
+        uint64  validSince;
+        uint64  validUntil;
```

Comment:
Could be optimized a little bit more: https://github.com/Loopring/protocols/issues/1579#issuecomment-670571212

Reply:
I have another idea, but let me do it in another PR

Reply:
Will do it in another PR as we also have to change UpgraderModule because the ABI of the previous SecuirtyStore and the new one are not compatible.

Reply:
Reduced the storage of each guardian from 116 bytes t0 29 bytes